File: ThiModule/ThiModule5/src/app/core/model/car.ts
import {CarType} from './car-type';

export interface Car {
  id: number,
  typeCar: CarType,
  name: string,
  startPlace: string,
  endPlace: string,
  phone: string,
  email: string,
  timeStart: string,
  timeEnd: string,
  deleted: boolean
}

File: ThiModule/WebserviceForThiModule5/src/main/java/com/codegym/model/entity/car/Car.java
package com.codegym.model.entity.car;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;

@Entity
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class Car {
    @Id
    private Integer id;

    @ManyToOne(targetEntity = CarType.class)
    @JoinColumn(name = "type_id", referencedColumnName = "id")
    private CarType type;
    private String name;
    private String startPlace;
    private String endPlace;
    private String phone;
    private String email;
    private String timeStart;
    private String timeEnd;
    private boolean isDeleted;
}